templates/BUILD.bazel

```diff
@@ -1,6 +1,12 @@
 # Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
+filegroup(
```

Comment:
There is no need for a file group, you can just export the file using `exports_files`, see https://docs.bazel.build/versions/master/be/functions.html#exports_files for details.